toolchain/check/handle_modifier.cpp

```diff
@@ -33,28 +34,16 @@ static auto EmitNotAllowedWithDiagnostic(Context& context,
       .Emit();
 }
 
-static auto GetAccessModifierEnum(Lex::TokenKind token_kind)
-    -> KeywordModifierSet {
-  switch (token_kind) {
-    case Lex::TokenKind::Private:
-      return KeywordModifierSet::Private;
-    case Lex::TokenKind::Protected:
-      return KeywordModifierSet::Protected;
-    default:
-      CARBON_FATAL() << "Unhandled access modifier keyword";
-  }
-}
-
-auto HandleAccessModifierKeyword(Context& context, Parse::NodeId parse_node)
-    -> bool {
-  auto keyword = GetAccessModifierEnum(
-      context.tokens().GetKind(context.parse_tree().node_token(parse_node)));
+auto HandleModifier(Context& context, Parse::NodeId parse_node,
```

Comment:
Shouldn't this have a name that doesn't start with `Handle` since it is not called by the same mechanism as the other `Handle` functions?

Reply:
It should definitely be static, that's a mistake. But there are a number of functions that are `static auto Handle` which are helpers for the handle function, e.g. `HandleStatementKeywordFinish` -- I think the `static` and different arguments help make it apparent that they're not part of the boilerplate handlers.